docs/deployment/configuration/resource_manager.rst

```diff
@@ -13,14 +13,6 @@ Flyte ResourceManager
 
 When using a plugin that connects to a platform with a robust resource scheduling mechanism, like the K8s plugin, we recommend leaving the default ``flyteresourcemanager`` configuration in place. However, with web API plugins (for example), the rate at which Flyte sends requests may overwhelm a service, and we recommend changing the ``resourcemanager`` configuration.
 
-
-The below attribute is configurable within FlytePropeller, which can be disabled with:
```

Comment:
any reason to delete this?

Reply:
It seemed duplicative of lines 7-12 above, but I can add it back if it seems helpful to call it out again with the emphasis on disabling resource manager.

Reply:
gotcha, ripping it out sounds good 👍